src/NuGet.Clients/NuGet.PackageManagement.UI/Models/PackageItemListViewModel.cs

```diff
@@ -272,29 +285,66 @@ public bool IsPackageDeprecated
         public Uri IconUrl { get; set; }
 
         public Lazy<Task<IEnumerable<VersionInfo>>> Versions { private get; set; }
+        public Task<IEnumerable<VersionInfo>> GetVersionsAsync() => (Versions ?? LazyEmptyVersionInfo).Value;
 
-        public Task<IEnumerable<VersionInfo>> GetVersionsAsync() => Versions.Value;
+        public Lazy<Task<PackageDeprecationMetadata>> DeprecationMetadata { private get; set; }
+        public Task<PackageDeprecationMetadata> GetPackageDeprecationMetadataAsync() => (DeprecationMetadata ?? LazyNullDeprecationMetadata).Value;
 
-        private Lazy<Task<NuGetVersion>> _backgroundLoader;
+        private Lazy<Task<NuGetVersion>> _backgroundLatestVersionLoader;
+        private Lazy<Task<PackageDeprecationMetadata>> _backgroundDeprecationMetadataLoader;
 
         private void TriggerStatusLoader()
         {
-            if (!_backgroundLoader.IsValueCreated)
+            if (!_backgroundLatestVersionLoader.IsValueCreated)
             {
-#pragma warning disable VSTHRD110 // Observe result of async calls
-                Task.Run(async () =>
-#pragma warning restore VSTHRD110 // Observe result of async calls
-                {
-                    var result = await _backgroundLoader.Value;
+                NuGetUIThreadHelper.JoinableTaskFactory
```

Comment:
Why is this being kicked off asynchronously?

Reply:
It calls `IPackageSearchMetadata.GetVersionsAsync`, which makes calls to registration in the installed tab case. The same applies to `_backgroundDeprecationMetadataLoader`.